src/mainwindow.cpp

```diff
@@ -405,7 +405,7 @@ bool MainWindow::LoadLogFile(QString path)
     QFileInfo fi(path);
     if (!fi.exists() || !fi.isFile())
     {
-        statusBar()->showMessage(QString("Unable to locate '%1'").arg(path), 10000);
```

Comment:
Happened to me when clicking in the menu on `Live Capture` -> `Open log.txt`. It tried to open log.txt which did not exist. At first, I thought the buttons were broken...

You are right with regards to the multi-file scenario. Bad idea to open multiple error boxes. Let's see if it is possible to show all error messages in a single dialog...

Reply:
For the multiple file, it is unlikely that anyone would get into the multiple error dialog scenario. Because to open multiple files you need to select them (Ctrl+click) from the file finder.

I guess someone could manually type a bunch of invalid paths too, but I find that veeery unlikely.
The scenario of someone clicking on the "Open log.txt" menu item with the file not existing is way more feasible. Getting that scenario right is more important.

Reply:
Hm... we could also just disable the "Open log.txt" from the menu, in case the corresponding file does not exist.
What do you think? Should we back out this change, i.e. keep displaying the message in the status bar, and just disable the menu entry if the file does not exist?

Reply:
This check is also used for "Open (beta) log.txt" and "Recent files", which is more likely to encounter this error for me. 
Disabling the open log menu item would be more complicated as we also need a way to monitor the file and re-enable it when the file is created again.
I think someone prefers the auto-dismiss error so you won't have to click a button to dismiss it every time. But as you noted, many people might not notice the error message in status bar. Ideally it could be an auto-dismiss dialog that's noticeable enough but yet not intrusive. But last time I did a quick 1-minute search, there isn't any in Qt.

Reply:
I guess that there are 3 options now:
1) Using a MessageBox. Blocking and noticeable
2) Using the Status Bar. Non-blocking but not noticeable
3) Disabling the menus. Avoids the bad behavior but might be tricky to implement.

I prefer keeping it simple and being noisy about warnings, so I vote option 1

Reply:
I would also vote for 1.
I checked if there is an easy way to collapse multiple popup messages into a single one but I did not see a possibility to achieve it without restructuring the code quite a bit
Hence, I would vote for merging this change as is if everyone agrees

Reply:
+1 on 1. Fine with 2.